tests/iso-8859-2-hungarian/auto-apro.hu.xml

```diff
@@ -1,474 +0,0 @@
-<?xml version="1.0" encoding="iso-8859-2" ?>
```

Comment:
I completely agree that the tag filtering is terribly broken.  You see, I only recently took over maintaining chardet about a year ago, and there are many things I haven't had the time to fix yet.  We had a discussion recently about how bizarre the tag filtering is in #46.  

Anyway, I think the real solution here would be to change the API such that there is an option `chardet.detect()` called `remove_tags` that would perform that tag filtering as a preprocessing step.  We want something that works for plaintext as well as HTML/XML files.